documentation/source/user_section/command-line.rst

```diff
@@ -135,176 +192,181 @@ sct_convert
 
 
 sct_create_mask
-==================
+===============
 
 .. program-output:: sct_create_mask -h
 
 
 sct_crop_image
-=================
+==============
 
 .. program-output:: sct_crop_image -h
 
 
-.. _sct_deepseg_gm:
+sct_deepseg
+==============
```

Comment:
Nitpick, but it seems this header also should also be updated.
```suggestion
===========
```

Reply:
fixed in bde6e598ede15e70bfd84e133228fef71d00d3f0

Reply:
Ah, it seems an extra "g" character was added in that commit!

Reply:
oopsi dc773aab5fceff541117dd5a851f6c9535ee1a09